### ISSUE TITLE
change page Indicator Image [SMPageControl] / EAIntroView in swift

### ISSUE THREAD

**Issue by Amaneo** [ealeksandrov/EAIntroView#161]
http://stackoverflow.com/q/36965228/4837354

**Comment by ealeksandrov** [comment]
Good catch. Try looking for custom page controls that are subclasses of UIPageControl.

I'll update demo when I have time.

**Comment by ealeksandrov** [comment]
After looking through this - there is no ability to effectively subclass UIPageControl so we need to ultimately choose some particular custom PageControl that will suit all our customization needs.

Since there is no good open-source solution (`SMPageControl` is not maintained) - I will return to choosing/creating dependency when rewriting EAIntroView in Swift.

For obj-c `SMPageControl` hack is ok but I don't recommend it for Swift. I won't add `SMPageControl` as dependency now as I don't think that it is needed for most of users and others can use current workaround.
